x/inflation/alias.go

```diff
@@ -0,0 +1,23 @@
+package inflation
+
```

Comment:
I've never really liked this alias pattern - it seems unnecessary, especially when there are `var`s.  It's okay to keep for now but we should do a pass to clean these up.